logos/src/logos/dbutils/dbmanager.py

```
@@ -0,0 +1,774 @@
+        result = self.session.execute(sql, {
+            "timestamp_request": datetime.datetime.now(datetime.timezone.utc),
+            "process_id": process_id,
+            "client_ip": client_ip if log_level == "FULL" else None,
+            "input_payload": input_payload if log_level == "FULL" else None,
+            "headers": headers if log_level == "FULL" else None,
+            "privacy_level": log_level
+        })
+
+        log_id = result.scalar()
+        self.session.commit()
+        return {"result": f"Created Token Type.", "log-id": log_id}, 200
```

Ruff (0.11.9)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)